src/Open3D/IO/ClassIO/PointCloudIO.cpp

```diff
@@ -90,8 +91,10 @@ bool ReadPointCloud(const std::string &filename,
         filename_ext = format;
     }
 
-    std::cout << "Format = " << format << std::endl;
-    std::cout << "Extension = " << filename_ext << std::endl;
+    if (!quiet) {
+        std::cout << "Format = " << format << std::endl;
```

Comment:
don't know why `std::cout` is here, let's use`utility::LogInfo`

Reply:
I switched it to LogDebug and removed quiet that I added